test/specs/fedprops/prefetching.ts

```diff
@@ -85,4 +85,60 @@ describe( 'Property Prefetching', function () {
 		// +1 for ?
 		assert.strictEqual( ( await $$( 'ul.special li' ) ).length, NUM_PROPERTIES + 3 );
 	} );
+
+	it( 'Should be able to change limit', async () => {
+		await browser.url(
+			`${testEnv.vars.WIKIBASE_URL}/wiki/Special:RecentChanges?limit=50&days=7&urlversion=2&enhanced=0`
+		);
```

Comment:
The thing I have in mind is that we make the base Page object look like this (maybe renaming to `MediaWikiPage` or `WikibasePage`):
```
export class Page {
	public async open( path: string ): Promise<void> {
		await browser.url( `${browser.options.baseUrl}/${path}` );
	}
}

export default new Page();
```
And then do:
```
Page.open( 'Special:RecentChanges?limit=50&days=7&urlversion=2&enhanced=0' )
```
In all the cases where we a unique Page object is not justified. 

I have a couple other thoughts in line with that more global chat about Page Object we touch-on the other day. I'm happy to get into further with you anytime you feel find the right moment for it 👌🏼

Reply:
If I understand correctly, that's basically what `Page` already does, and we could fully use `page.Open('Special:RecentChanges')` now. That, and a few other things, I'd like to implement in a story to clean up this code some more.